packages/design-system/src/components/Table/Table.stories.tsx

```diff
@@ -40,7 +40,7 @@ const tableTemplateData = {
         'Statement adopted by the Continental Congress declaring independence from the British Empire.',
       links: (
         <a href="https://billofrightsinstitute.org/founding-documents/declaration-of-independence/">
-          https://billofrightsinstitute.org/founding-documents/declaration-of-independence/
+          Declaration of Independence
```

Comment:
I thought our table styles weren't responsive but discovered the issue was in the text string didn't break correctly when a naked link was used. Rather than adding styles to fix this example, I fixed the example to work with our styles.